tests/cleaning/quality_filter_test.py

```diff
@@ -0,0 +1,218 @@
+from dfm.cleaning import QualityFilter
+import pytest
+
+
+class TestQualityFilter:
+    """Unit tests for the QualityFilter class"""
+
+    @pytest.fixture(scope="class")
+    def tweet_texts(self):
+        return [
+            "jeg er glad",
+            "56789okd23456789098765sds",
+            "jeg er glad...",
+            "jeg er glad…",
+            "67 54 13 B7",
+            "#yolo # test ##",
+        ]
+
+    @pytest.fixture(scope="class")
+    def long_text(self):
```

Comment:
```suggestion
    def valid_text(self):
```
Is probably more descriptive